ajax/libs/jmespath/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jmespath",
+  "version": "0.15.0",
+  "filename": "jmespath.min.js",
+  "description": "JMESPath implementation in JavaScript",
+  "homepage": "http://jmespath.org/",
+  "keywords": [
+    "jmespath",
+    "jsonpath",
+    "json",
+    "xpath"
```

Comment:
This is related XML but this library is a query language for JSON. It might be not suitable.